### DISCUSSION TITLE
How to report vulnerabilities on GitHub

### DISCUSSION THREAD

**Discussion by Alpop12** [community/community#147123] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Issues

### Body

How to report vulnerabilities on GitHub

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Alpop12 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you, **General.** Good luck!

**Reply by Alpop12** [reply]
I hope to get the reward

**Comment by Alpop12** [comment]
I hope to get the reward
